eZ/Publish/API/Repository/Values/Content/Query/Criterion/DateMetadata.php

```diff
@@ -42,18 +42,24 @@ class DateMetadata extends Criterion
      */
     public const CREATED = 'created';
 
+    /**
+     * DateMetadata target: move to trash date
+     * Only for LSE.
+     */
+    public const TRASHED = 'trashed';
+
     /**
      * Creates a new DateMetadata criterion on $metadata.
      *
      * @throws \InvalidArgumentException If target is unknown
      *
-     * @param string $target One of DateMetadata::CREATED or DateMetadata::MODIFIED
+     * @param string $target One of DateMetadata::CREATED, DateMetadata::MODIFIED or DateMetadata::TRASHED (works only with LSE)
```

Comment:
This is not true, LSE does not cover searching in Trash at all as no Search engine does. Conceptually search in trash is not a part of LSE.
What could be safer is:
```suggestion
     * @param string $target One of DateMetadata::CREATED, DateMetadata::MODIFIED or DateMetadata::TRASHED (applies to TrashService::findTrashItems only).
     *
     * @see \eZ\Publish\API\Repository\TrashService::findTrashItems   
```